### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by delphiactual**
This PR renames all `*.template.html` to just `*.html` and moves all internal templates to external templates with associated import statements.

#1401

### CODE REVIEW

**Comment by bhollis** [comment]
Thanks, this is great. Does this need anything more for the reviews stuff?

**Comment by delphiactual** [comment]
nope ready to go for review stuff

File: src/scripts/store/dimStoreBucket.directive.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import angular from 'angular';
 import _ from 'underscore';
 import template from './dimStoreBucket.directive.html';
+import template2 from './dimStoreBucket.directive2.html';
```
Comment (bhollis) [inline src/scripts/store/dimStoreBucket.directive.js]: I've been calling this `dialogTemplate` or `moveDialogTemplate`.

File: src/scripts/move-popup/dimMoveItemProperties.directive.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import angular from 'angular';
 import _ from 'underscore';
 import template from './dimMoveItemProperties.directive.html';
+import template2 from './dimMoveItemProperties.directive2.html';
```
Comment (bhollis) [inline src/scripts/move-popup/dimMoveItemProperties.directive.js]: Ditto here, we can come up with a better name than "template2"

File: src/scripts/move-popup/dimMoveItemProperties.directive.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import angular from 'angular';
 import _ from 'underscore';
 import template from './dimMoveItemProperties.directive.html';
-import template2 from './dimMoveItemProperties.directive2.html';
+import moveDialogTemplate from './dimMoveItemProperties.directive2.html';
```
Comment (bhollis) [inline src/scripts/move-popup/dimMoveItemProperties.directive.js]: Can we fix the filenames too?

Comment (delphiactual) [inline src/scripts/move-popup/dimMoveItemProperties.directive.js]: fixed

**Pull request merged**